spec/factories/participants.rb

```diff
@@ -4,4 +4,14 @@
     sequence :number
     bus
   end
+
+  trait :total_score do
```

Comment:
Hmm, what's the purpose of this trait? I'm not clear on that from reading the name.